### DISCUSSION TITLE
variable olive row widths

### DISCUSSION THREAD

**Discussion by theIZI** [Courseplay/Courseplay_FS22#3259] [Q&A]
hi. is there away for course generator to see different widths. i use the variable mod bc the original olive rows are too narrow. i just made a huge field with olives just to realize the coursegenerator cant make course ;);)

**Comment by Tensuko** [comment]
There is no way to do that.
It might be that we implement this, when the new course generator is there and then the vine generator also gets a rework.
But for now -> stick to Giants standarts.
